packages/client/tsconfig.eslint.json

```diff
@@ -1,14 +1,5 @@
 {
   "extends": "./tsconfig.json",
```

Comment:
I don't have context on why a separate `tsconfig.eslint.json` is used here and in `server`. Typically, you just have a single `tsconfig.json` that's used throughout.